docs/content/overview/cdc-pipeline.md

```diff
@@ -0,0 +1,107 @@
+
+### Table ID
+When connecting to external systems, it is necessary to establish a mapping relationship with the storage objects of the external system. This is what Table Id refers to.
+
+To be compatible with most external systems, the Table ID is represented by a 3-tuple, (namespace, schemaName, table). Connectors need to establish the mapping between Table ID and storage objects in external systems. For instance, a database table in MySQL/Doris is mapped to (null, database, table), and a topic in a message queue system such as Kafka is mapped to (null, null, topic).
+
+### Data Source
+Data Source is used to access metadata and read the changed data from external systems.
+A Data Source can read data from multiple tables simultaneously.
+
+To describe a data source, the follows are required:
+* Type: The type of the source, such as MySQL, Postgres.
+* Name: The name of the source, which is user-defined (optional, with a default value provided).
+* Other custom configurations for the source.
+
+
+For examle, we can use yaml to define a mysql source:
+```yaml
+source:
+  type: mysql
+  name: mysql-source   #optional，description information
+  host: localhost
+  port: 3306
+  username: admin
+  password: pass
+  tables: adb.*, bdb.user_table_[0-9]+, [app|web]_order_.*
+```
+The ability to synchronize the entire library comes from the implementation of Flink source. After obtaining the user configuration, the Flink source connector needs to be able to send all data in a database downstream according to the specified data type.
+
+### Data Sink
+The Data Sink is used to modify metadata and write change data to external systems. A Data Sink can write to multiple tables simultaneously.
+
+To describe a data sink, the follows are required:
+* Type: The type of the sink, such as MySQL or PostgreSQL.
+* Name: The name of the sink, which is user-defined (optional, with a default value provided).
+* Other custom configurations for the sink.
+
+
+For examle, we can use yaml to define a kafka sink:
+```yaml
+sink:
+  type: kafka
+  name: mysink-queue           	# Optional parameter for description purpose
+  bootstrap-servers: localhost:9092
+  auto-create-table: true      	# Optional parameter for advanced functionalities
+```
+
+### Route
+Routing, for each change event, specifies the target table ID of the event. The most typical application scenario is the merge of sub-databases and sub-tables, routing multiple upstream source tables to the same result table.
+
+To describe a route, the follows are required:
+* source-table: source table id, supports regular expressions
+* sink-table: Result table id, supports regular expressions
+* description: Routing rule description, user-defined (optional, default value provided)
+
+For example, if synchronize the table named "web_order" in the database named "mydb" to a Kafka topic named "ods_web_order", we can use yaml to define a route：
```

Comment:
For example, if synchronize the table 'web_order' in the database 'mydb' to a Kafka topic 'ods_web_order', we can use this `yaml` file to define this route：